### ISSUE TITLE
Controllers not showing up when reactivating them

### ISSUE THREAD

**Issue by jnk22** [Ryochan7/DS4Windows#235]
Hi,

I have a problem with my new DS4 controllers when using DS4Windows.

Setting them up and everything works perfectly fine, but when setting an "Idle Disconnect" or after turning them off manually, DS4Windows won't show them when I try to enable them again (by pressing the PS button).

Log for that error:
```
10.03.2018 14:10:04: Found Controller:F4:93:9F:C9:98:25 (BT)
10.03.2018 14:10:04: Plugging in X360 Controller #1
10.03.2018 14:10:04: X360 Controller # 1 connected
10.03.2018 14:10:04: Controller 1 is using Profile “Default"
10.03.2018 14:11:04: F4:93:9F:C9:98:25 disconnecting due to idle disconnect
10.03.2018 14:11:04: X360 Controller # 1 unplugged
10.03.2018 14:11:04: Controller 1 was removed or lost connection
10.03.2018 14:11:23: Found Controller:F4:93:9F:C9:98:25 (BT)
10.03.2018 14:11:23: Plugging in X360 Controller #1
10.03.2018 14:11:23: X360 Controller # 1 connected
10.03.2018 14:11:23: Controller 1 is using Profile “Default"
```

This is how it looks like after pressing the PS button (last line from log):
![log_error](https://user-images.githubusercontent.com/24391401/37242642-534375e6-246d-11e8-8337-7891020ce3d1.jpg)

So it seems that it actually finds the controller, but won't show it. The lightbar only stays lightblue instead of the default color (darkblue), and the touchpad doesn't work anymore. The buttons still seem to work in some applications though.

The only way to use a controller again is to remove them from Windows devices and readd them.
Is there any way so I can turn them off, without the need of repairing them every time?

I have tested this on 2 Windows 8.1 machines with 2 different DS4 controllers and several DS4Windows versions (1.4.100-113)

**Comment by Ryochan7** [comment]
It really seems like DS4Windows is not reading the input reports from the controller. I guess maybe the byte indicating the report type might not be set as expected so the report is ignored. I have tested Idle Disconnect recently and it worked fine on my system. How are you manually turning off the controller: Disconnect BT special action, right click BT icon, holding PS button for several seconds, etc? Some applications, like Steam, might have access to the DirectInput device so that might be why some buttons still work in other applications.

**Comment by jnk22** [comment]
I "soft disconnect" them using the PS+Options button (so the light turns off).

It seems like it really must be a software or driver problem on this one machine I am using.

Both controllers work fine on the other one now (Windows 8.1, laptop with integrated bluetooth), so one problem is gone already.
I have also tested the controllers on another machine (Windows 10 with an ASUS USB-BT400). Everything is working as it should be there.

So only this one computer (Windows 8.1, ASUS USB-BT400) does not work properly for now.
I still have some Logitech F710 (Xinput) and Logitech Rumblepad 2 drivers installed and also Kodi (media software) installed. I am going to delete the drivers and try to get it working. I'll report back.

**Comment by jnk22** [comment]
I have done some further testing, unfortunately I could not get it to work.

I uninstalled all Logitech drivers, removed all my dongles for my remote control, Logitech mouse and keyboard, made sure that Kodi would not start at boot and also reinstalled the DS4 drivers and reinstalled DS4Windows with a fresh profile folder.

I also tried various settings in DS4Windows, like "Disconnect from BT when Stopping" and "Hide DS4 Controller". Same problem still persists.

So every time I "turn off" my DS4 controller, I can only get Windows to recognize it, but not DS4Windows. So even though the default profile gets selected, I can not see it in DS4Windows and can not use the touchpad or anything.

Is there anything else that I could try?
